source/_posts/2024-09-04-release-20249.markdown

```diff
@@ -108,27 +67,47 @@ Cards and sections will continue to follow the [Z-Grid rules](/blog/2024/03/04/d
 
 When switching to smaller screens, these larger sections will revert to one section wide (if this causes issues for certain big cards, we would recommend using our device [visibility settings](/blog/2024/06/05/release-20246/#control-visibility-of-cards)).
 
+<img src="/images/blog/2024-09/section-settings.png" alt="Screenshot of the section settings">
+
 ### Full span cards
 
 Larger sections mean larger cards. Instead of just stretching out the layout of a card, there is also a new option on cards to have them always use the full width of the section.
 
+### Badges
+
+Talking about dashboards, the newly styled badges we [introduced](/blog/2024/08/07/release-20248/#new-badges-for-your-dashboard) in last release, are now more configurable.
+
+Instead of a display mode, you can decide yourself what element you want to show. This allows you for example to hide the icon of an badge and only show the state of the device you want to show in your badge.
```

Comment:
```suggestion
Instead of using display mode, you can decide for yourself what element you want to show. This allows you, for example, to hide the icon and only show the state of the device in your badge.
```